src/features/settings/components/SettingTabContainer/index.tsx

```
@@ -0,0 +1,228 @@
+			);
+			if (!selected) {
+				settings.pineconeIndexName = pineconeIndexes[0].name;
+				plugin.saveSettings();
+			}
+			setPineconeIndex(settings.pineconeIndexName);
+		}
+	}, [pineconeIndexes]);
+
+	useEffect(() => {
+		if (isError) {
+			console.error("Failed to fetch indexes:", error);
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: error

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)